cdap-gateway/src/main/java/co/cask/cdap/gateway/router/handlers/SecurityAuthenticationHttpHandler.java

```diff
@@ -80,6 +83,23 @@ public SecurityAuthenticationHttpHandler(String realm, TokenValidator tokenValid
     this.discoveryServiceClient = discoveryServiceClient;
     this.discoverables = discoveryServiceClient.discover(Constants.Service.EXTERNAL_AUTHENTICATION);
     this.configuration = configuration;
+    this.bypassPattern = createMatcher(configuration.get(Constants.Security.Router.BYPASS_AUTHENTICATION_REGEX));
```

Comment:
should this be stored in cdap-security.xml, instead of cdap-site.xml?
So SConfiguration, instead of CConfiguration?